Description: embeds voms-api-java 2.0.8 in the lib directory.
 EMI voms-api-java 2.0.8 does not have a Debian package (yet).
 The assembly embeds voms-api-java 2.0.8 from the SWITCH maven repo into
 the lib directory.
Author: Valery Tschopp <valery.tschopp@switch.ch>

--- a/src/main/assembly/fakeroot.xml
+++ b/src/main/assembly/fakeroot.xml
@@ -34,7 +34,6 @@
                 <exclude>net.jcip:*</exclude>
                 <exclude>log4j:log4j</exclude>
                 <!-- use jars from /usr/share/java directory -->
-                <exclude>org.glite.voms:voms-api-java</exclude>
                 <exclude>org.glite.authz:pdp-pep-common</exclude>
             </excludes>
             <fileMode>0644</fileMode>
